lib/helper.ts

```diff
@@ -43,22 +43,31 @@ export interface FileStream extends NodeJS.ReadableStream {
 }
 
 // custom type guards
-function isFileObject(obj: any): obj is FileObject {
+export function isFileObject(obj: any): obj is FileObject {
   return Boolean(obj && obj.value);
 }
 
 function isFileStream(obj: any): obj is FileStream {
   return obj && isReadable(obj) && obj.path;
 }
 
+export function isFileParamAttributes(obj: any): obj is FileParamAttributes {
+  return obj && obj.data &&
```

Comment:
I don't think this is a big deal but this should maybe be wrapped in a `Boolean()` cast to ensure the result is either `true` or `false`. The way is stands, it could end up returning `undefined` if `obj` or `obj.data` is undefined. That is fine, because it still evaluates to falsy, but it is probably more clear to force the boolean type.